pkg/remediator/watch/filteredwatcher.go

```diff
@@ -461,7 +461,7 @@ func (w *filteredWatcher) shouldProcess(object client.Object) bool {
 
 	desiredManager := declared.ResourceManager(w.scope, w.syncName)
 	conflictErr := status.ManagementConflictErrorWrap(object, desiredManager)
-	currentManager := conflictErr.ConflictingManager()
+	currentManager := conflictErr.CurrentManager()
 
 	// TODO: Move logging & metric recording into the conflict handler
```

Comment:
Can this be part be consolidated as well?

Reply:
consolidated?

Reply:
Change the below section to use conflict.Record?

Reply:
Good catch. Fixed.